protocol/avdl/chat1/local.avdl

```diff
@@ -256,8 +258,22 @@ protocol local {
     string senderUsername;
     string senderDeviceName;
     string senderDeviceType;
+
+    // MessageBoxed.V1: Hash of the encrypted header ciphertext.
+    // MessageBoxed.V2: Hash of MessageBoxed.headerSealed (.v || .n || .b)
     Hash headerHash;
+
+    // TOOD Maybe get rid of this field in favor of verificationKey.
+    //      If so, bump-nuke the caches in storage_blockengine and any other persistent
+    //      storage of MessageUnboxedValid.
+    // MessageBoxed.V1: Header signature. Included for the verification key.
+    // MessageBoxed.V2: Null
     union {null, SignatureInfo} headerSignature;
+
+    // MessageBoxed.V1: Null
+    // MessageBoxed.V2: The verification key used to unbox.
+    union {null, bytes} verificationKey;
```

Comment:
Sure: It's the public half of the device signing key that is used to open the `headerSealed` (`SignEncrypted`) header. It is that which used to be `HeaderPlaintextV1.HeaderSignature.K` (`SignatureInfo.K`) in the V1 `HeaderSignature`. It's the key that signs the message and is asserted to belong to the sender.

I put it here and didn't include it in `SignEncryptedInfo` but could be talked into doing that instead easily. It seems sort of good to me for the program to have to explicitly get the verification key out to facilitate remembering to validate that it belongs to the correct person.

Also, it may or may not be a good idea to duplicate it inside the header. To protect against non-exclusive ownership problems. But I'm not sure about that cc @oconnor663 

Should some of that go in a comment?